### ISSUE TITLE
How to properly check PSR-0/4?

### ISSUE THREAD

**Issue by CZechBoY** [slevomat/coding-standard#331]
Hi,
I found that this setting does not work as I expect.
```xml
     <rule ref="SlevomatCodingStandard.Files.TypeNameMatchesFileName">
        <properties>
            <property name="rootNamespaces" type="array" value="/=>App"/>
        </properties>
    </rule>
```

Can anybody tell me how to set it correctly?

I run CS with this command: `php vendor/squizlabs/php_codesniffer/bin/phpcs -d memory_limit=1G --standard=ruleset.xml --extensions=php --encoding=utf-8 --tab-width=4 -sp ../../../web/App`
directory structure:
```
- other
-- Tests
--- cs
---- vendor
----- slevomat
- web
-- App
--- bootstrap.php
--- other php sources
```

Every file inside App directory is marked as bad. How to set it properly?

Thanks

**Comment by CZechBoY** [comment]
It should be `App=>App`
